platform/mellanox/mlnx-platform-api/sonic_platform/module.py

```diff
@@ -247,3 +249,182 @@ def get_sfp(self, index):
         with self.lock:
             self.initialize_single_sfp(index)
             return super(Module, self).get_sfp(index)
+
+
+class DpuModule(ModuleBase):
```

Comment:
@gpunathilell,

I don't see here implementation for is_midplane_reachable, and get_midplane_ip

Reply:
`is_midplane_reachable` and `get_midplane_ip` to be added back to the implementation

Reply:
Done